src/client/ui/status-bar/index.js

```diff
@@ -113,6 +130,40 @@ export default class StatusBar {
         this.fixtureContainer.appendChild(userAgentDiv);
     }
 
+    _createUnlockPageArea (container) {
+        var unlockPageArea      = document.createElement('div');
+        var unlockPageContainer = document.createElement('div');
+        var unlockIcon          = document.createElement('div');
+        var unlockText          = document.createElement('span');
+
+        unlockText.textContent = UNLOCK_PAGE_TEXT;
+
+        shadowUI.addClass(unlockPageArea, UNLOCK_PAGE_AREA_CLASS);
+        shadowUI.addClass(unlockPageContainer, UNLOCK_PAGE_CONTAINER_CLASS);
+        shadowUI.addClass(unlockPageContainer, LOCKED_CLASS);
+        shadowUI.addClass(unlockIcon, UNLOCK_ICON_CLASS);
+
+        if (browserUtils.isMacPlatform)
+            unlockPageContainer.style.paddingTop = '3px';
+
+        container.appendChild(unlockPageArea);
+        unlockPageArea.appendChild(unlockPageContainer);
+        unlockPageContainer.appendChild(unlockIcon);
+        unlockPageContainer.appendChild(unlockText);
+
+        this._bindClickOnce([unlockPageContainer], () => {
+            shadowUI.removeClass(unlockPageContainer, LOCKED_CLASS);
+            shadowUI.addClass(unlockPageContainer, UNLOCKED_CLASS);
+            unlockText.textContent = PAGE_UNLOCKED_TEXT;
+
+            this.emit(this.UNLOCK_PAGE_BTN_CLICK, {});
+        });
+
+        unlockPageArea.style.display = 'none';
```

Comment:
Can we move it (and [this](https://github.com/DevExpress/testcafe/pull/1849/files#diff-2554eb3e168f40fd07e2bd9e3e175644R195)) in less? Or we need to set `display` here because of size calculation?